packages/plugin-core/src/commands/GotoNote.ts

```diff
@@ -274,13 +275,35 @@ export class GotoNoteCommand extends BasicCommand<
     let pos: undefined | Position;
     const out = await this.extension.pauseWatchers<GoToNoteCommandOutput>(
       async () => {
+        // First query to see if the note exists to determine if we will send
+        // telemetry, createIfNew set to false
+        const respFromExistenceCheck = await client.getNoteByPath({
+          npath: qs,
+          createIfNew: false,
+          vault,
+          overrides,
+        });
+
         const { data } = await client.getNoteByPath({
           npath: qs,
           createIfNew: true,
           vault,
           overrides,
         });
         const note = data?.note as NoteProps;
+
+        // If a new note is going to be created, check if we should send meeting
+        // note telemetry.
+        if (!respFromExistenceCheck.data?.note) {
```

Comment:
you should be able to get this from the `changed` property of `getNoteByPath`

_.filter(changed, ent => ent.status === "created" && note.fname = qs)